tests/python/pants_test/integration/log_output_integration_test.py

```diff
@@ -31,3 +31,31 @@ def add(x: int, y: int) -> int:
     result.assert_success()
     assert "[DEBUG] Starting: Run MyPy on" in result.stderr
     assert "[DEBUG] Completed: Run MyPy on" in result.stderr
+
+
+def test_log_filtering_by_target() -> None:
+    sources = {
+        "src/python/project/__init__.py": "",
+        "src/python/project/lib.py": dedent(
+            """\
+            def add(x: int, y: int) -> int:
+                return x + y
+            """
+        ),
+        "src/python/project/BUILD": "python_library()",
+    }
```

Comment:
Rather than copying and pasting, can you please either use a `@pytest.fixture` or make this value be a module-level constant?

Reply:
Actually, I don't think there's any need for this source file, if all you're running is `list`. It's fine to have no tmpdir afaict.